files/shaders/lighting.glsl

```diff
@@ -1,6 +1,6 @@
 #define MAX_LIGHTS 8
 
-vec4 doLighting(vec3 viewPos, vec3 viewNormal, vec4 vertexColor)
+vec4 doLighting(vec3 viewPos, vec3 viewNormal, vec4 vertexColor,float shadow)
```

Comment:
This approach won't work without per-pixel lighting, which we recommend against as Morrowind's assets often look wrong with it (thanks, Todd Howard). Also, while better in theory, it's still being applied to all the lights, not just the sun light, and that's not good as Morrowind uses negative-intensity lights to simulate ambient occlusion, and this can lead to bright shadows. It's also not being applied to the specular component of any lights.